public/app/config/channels.js

```diff
@@ -52,6 +52,8 @@ module.exports = {
   SET_USER_MODE_CHANNEL: 'user:mode:set',
   SET_SPACE_AS_FAVORITE_CHANNEL: 'user:set-space-favorite:set',
   SET_SPACE_AS_RECENT_CHANNEL: 'user:set-space-recent:set',
+  SET_ACTION_ACCESSIBILITY_CHANNEL: 'user:action:accessibility:set',
+  SET_ACTION_ENABLED_CHANNEL: 'user:action:enabled:set',
```

Comment:
I think for this one we could be more explicitly and use `SET_ACTIONS_AS_ENABLED`. Because it affects all actions, not just one. Or something completely different, such as `TOGGLE_ACTION_TRACKING` or `TOGGLE_ACTION_RECORDING`. What are your thoughts?